src/data/contributors/ShubhamRattra.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Shubham Rattra",
+    "skills" : ["Adroid development", "Java", "Kotlin"],
+    "cause" : "I code for the community",
```

Comment:
**I code for** is already part of string. 

Additionally, please write something unique to you as cause.

Reply:
I have written the community. Or do I have to write something else?

Reply:
You can write anything. you can take hint from the readme.